packages/api/codegen.yml

```diff
@@ -0,0 +1,35 @@
+#schema: ${API_URL}/admin
```

Comment:
is this needed, or a case when this comes handy?

Suggesting comment inside the comment if can be handy, if not removal)

Reply:
i removed the line. Thanks